recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
```

Comment:
I don't think there is support for multiple URLs. Are both of these needed or will one of them be sufficient? If both are needed, the build will need to download and decompress it manually.

Reply:
Oh there is. All are tried in this order. So if the package at some point gets archived we have a fallback URL. In BioConda we include 3 URLs. The third one our own archive URL.

Reply:
Weird. There was discussion about having multiple URLs that all got downloaded at once, but I don't recall whatever came of that (if anything?). Fallback URLs are a very different idea. Is that intentional by `conda-build` or does this just happen to work?

Reply:
In any event, I would suggest picking one and seeing if that fixes the issue.

Reply:
Only the first one is working today, but tomorrow only the second one can work.
This is what all 500+ r-* packages are using https://github.com/conda-forge/r-dplyr-feedstock/blob/master/recipe/meta.yaml#L12 and we never had a problem with it.

Reply:
Ok. It's only a suggestion to try to solve the problem. Even if we do find it is needed at `staged-recipes`, it can be reverted in the feedstock.